packages/blade/src/components/Indicator/_decisions/decisions.md

```diff
@@ -65,7 +65,7 @@ Sample usage:
 Web:
 
 - The component accepts an `accessibilityLabel` prop to let users pass an `aria-label` used by screen readers. This is helpful in building UIs where an `Indicator` is used without a visible `label`.
-- If `label` is provided without any `accessibilityLabel` we use an `aria-labelledby` internally to let the screen readers use `label` for a11y purposes.
+- If `label` is provided without any `accessibilityLabel` we use an `aria-label` internally to let the screen readers use `label` for a11y purposes.
```

Comment:
shouldn't this be `children` instead of `label` since that is the prop we pass? 

maybe something like

If `children` i.e label is provided without any `accessibilityLabel` we use an `aria-label` internally to let the screen readers use that for a11y purposes.

Reply:
Right, missed during update

Reply:
Fixed in https://github.com/razorpay/blade/pull/814/commits/69498410036c299fc6f507af70adecddd4228364